packages/icons/package.json

```
@@ -1,6 +1,6 @@
 {
 	"name": "@wordpress/icons",
-	"version": "1.3.1",
+	"version": "2.0.0",
```

Also, this shouldn't be changed until release, so you can leave it out.

```suggestion
	"version": "1.3.1",
```